api_server.py

```
@@ -81,12 +81,23 @@ def get_crew_list() -> List[Dict]:
+        if table == "crew_availability":
+            cursor.execute(
+                "SELECT end_time FROM crew_availability WHERE crew_id = ? AND start_time <= ? AND end_time > ? LIMIT 1",
+                (entity_id, now, now),
+            )
+        elif table == "appliance_availability":
+            cursor.execute(
+                "SELECT end_time FROM appliance_availability WHERE appliance_id = ? AND start_time <= ? AND end_time > ? LIMIT 1",
+                (entity_id, now, now),
+            )
+        else:
+            raise ValueError("Invalid table name")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)